GETTING_STARTED.md

```diff
@@ -0,0 +1,333 @@
+
+```
+NAME                   STATUS   ROLES         AGE     VERSION
+k3d-adelphi-server-0   Ready    etcd,master   10m     v1.19.3+k3s2
+k3d-adelphi-server-1   Ready    etcd,master   10m     v1.19.3+k3s2
+k3d-adelphi-server-2   Ready    etcd,master   9m51s   v1.19.3+k3s2
+```
+
+When you are done with the cluster, it can be deleted with:
+
+```
+k3d cluster delete adelphi
+```
+
+## Running Adelphi
+
+With helm installed, a kubernetes cluster deployed, and the kubectl context set appropriately you're now ready to deploy Adelphi.
+
+### Download Source
+
+To get started, you'll need a copy of the Adelphi repo to get access to the helm charts and components.
+
+You can clone it from [GitHub](https://github.com/datastax/adelphi) using [git](https://git-scm.com/):
+
+```
+git clone https://github.com/datastax/adelphi
+```
+
+Or download it manually from [GitHub](https://github.com/datastax/adelphi/archive/master.zip) and extract the contents of the archive.
+
+### Deployment
+
+As mentioned, Adelphi is deployed using helm.  Within the Adelphi repository, you will find the primary Adelphi chart and a series of templates used by that chart in the `/helm` directory.  
+
+To deploy Adelphi with the default configuration navigate to the source directory and run:
+
+```
+helm install adelphi helm/adelphi -n cass-operator
+```
+
+> :warning: The command shown above will install the Adelphi components into the `cass-operator` namespace.  Using this namespace is currently **required**.  
+>
+>Please see [#93](https://github.com/datastax/adelphi/issues/93) for more information and follow along as we work to better support custom namespace scoped installations.
+
+This will deploy Adelphi with the default [values](helm/adelphi/values.yaml) configuration.
+
+### Configuration
+
+To control the deployment configuration, many options are made available via the helm charts included with Adelphi.
+
+To learn more about setting values via helm, see their [documentation](https://helm.sh/docs/chart_template_guide/values_files/).
+
+In general, with helm, there are two ways to modify the values provided to a deployment:
+* via the command line using `--set value.path=value`
+* via a `yaml` file via `-f custom.values.yaml`
+
+#### Storage Classes
+
+The default `storageClass` configured to be used by Adelphi is tailored for deployment via k3d, `local-path`.  This storage class may not be available on all clusters and should be modified as necessary.
+
+For example, when deploying to a cluster within [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) it's necessary to customize the `storageClass` to be used.  An example of doing this is provided in [gke-values.yaml](helm/adelphi/gke-values.yaml) where `storageClass` is set to `standard`.
+
+This can also be applied via the helm CLI with:
+
+```
+helm install adelphi helm/adelphi --namespace cass-operator --set storageClass=standard
```

Comment:
I was actually hoping people would use the `gke-values.yaml` file directly with the `-f` param, such that they don't even have to worry about storage classes mean.
Another difference in that custom yaml is the cluster size, by default you have only one C* node per cluster, but with the custom gke values you'll have 3 (which is kind of required for real testing). So instead of setting yet another param in the command-line, we can just point to the file.

Anyway, no strong opinion here on the doc, but maybe we can at least add the alternative below:

```
helm install adelphi helm/adelphi -n cass-operator -f helm/adelphi/gke-values.yaml
```

Reply:
Ah, OK, we can do that.

Reply:
I added an example like you suggested, but did it up above in the basic deployment section, which makes it a bit more prominent.